dev-util/lldb/lldb-9999.ebuild

```diff
@@ -67,10 +71,10 @@ src_configure() {
 
 		-DLLVM_BUILD_TESTS=$(usex test)
 		# compilers for lit tests
-		-DLLDB_TEST_C_COMPILER="${EPREFIX}/usr/bin/clang"
-		-DLLDB_TEST_CXX_COMPILER="${EPREFIX}/usr/bin/clang++"
+		-DLLDB_TEST_C_COMPILER="$(type -P clang)"
+		-DLLDB_TEST_CXX_COMPILER="$(type -P clang++)"
```

Comment:
tc-get* stuff doesn't really return full paths. To be honest, it may not be strictly necessary here even but I don't really want to spend the time trying that. Especially with tests randomly failing thanks to vapier's ugly binutils-libs hackery.